### PR TITLE
[zh]Resync concepts configuration overview file

### PR DESCRIPTION
**Pull request by mengjiao-liu**
[zh]Resync concepts configuration overview file
en page: https://kubernetes.io/docs/concepts/configuration/overview/

related en PR: https://github.com/kubernetes/website/pull/29345

### CODE REVIEW

File: content/zh/docs/concepts/configuration/overview.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ This is a living document. If you think of something that is not on this list bu
 - 使用 YAML 而不是 JSON 编写配置文件。虽然这些格式几乎可以在所有场景中互换使用，但 YAML 往往更加用户友好。
 
 <!--
-- Group related objects into a single file whenever it makes sense. One file is often easier to manage than several. See the [guestbook-all-in-one.yaml](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/guestbook/all-in-one/guestbook-all-in-one.yaml) file as an example of this syntax.
+- Group related objects into a single file whenever it makes sense. One file is often easier to manage than several. See the [guestbook-all-in-one.yaml](https://github.com/kubernetes/examples/tree/master/guestbook/all-in-one/guestbook-all-in-one.yaml) file as an example of this syntax.
 -->
 - 只要有意义，就将相关对象分组到一个文件中。
  一个文件通常比几个文件更容易管理。
- 请参阅[guestbook-all-in-one.yaml](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/guestbook/all-in-one/guestbook-all-in-one.yaml) 文件作为此语法的示例。
+ 请参阅[guestbook-all-in-one.yaml](https://github.com/kubernetes/examples/tree/master/guestbook/all-in-one/guestbook-all-in-one.yaml) 文件作为此语法的示例。
```
Comment (tengqm) [inline content/zh/docs/concepts/configuration/overview.md:52]:
```suggestion
 请参阅 [guestbook-all-in-one.yaml](https://github.com/kubernetes/examples/tree/master/guestbook/all-in-one/guestbook-all-in-one.yaml)
 文件作为此语法的示例。
```

File: content/zh/docs/concepts/configuration/overview.md [hunk 2]
```diff
@@ -163,13 +163,13 @@ services) (which have a `ClusterIP` of `None`) for service discovery when you do
...
+- Define and use [labels](/docs/concepts/overview/working-with-objects/labels/) that identify __semantic attributes__ of your application or Deployment, such as `{ app: myapp, tier: frontend, phase: test, deployment: v3 }`. You can use these labels to select the appropriate Pods for other resources; for example, a Service that selects all `tier: frontend` Pods, or all `phase: test` components of `app: myapp`. See the [guestbook](https://github.com/kubernetes/examples/tree/master/guestbook/) app for examples of this approach.
 -->
 - 定义并使用[标签](/zh/docs/concepts/overview/working-with-objects/labels/)来识别应用程序
   或 Deployment 的 __语义属性__，例如`{ app: myapp, tier: frontend, phase: test, deployment: v3 }`。
   你可以使用这些标签为其他资源选择合适的 Pod；
   例如，一个选择所有 `tier: frontend` Pod 的服务，或者 `app: myapp` 的所有 `phase: test` 组件。
-  有关此方法的示例，请参阅[guestbook](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/guestbook/) 。
+  有关此方法的示例，请参阅[guestbook](https://github.com/kubernetes/examples/tree/master/guestbook/) 。
```
Comment (tengqm) [inline content/zh/docs/concepts/configuration/overview.md:172]:
```suggestion
  有关此方法的示例，请参阅 [guestbook](https://github.com/kubernetes/examples/tree/master/guestbook/)。
```

File: content/zh/docs/concepts/configuration/overview.md (hunk 1 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/configuration/overview.md]: Thanks for catch! Updated.

File: content/zh/docs/concepts/configuration/overview.md (hunk 2 above)
Comment (mengjiao-liu) [reply content/zh/docs/concepts/configuration/overview.md]: Updated.

**Comment by tengqm** [comment]
Minor nits: please wrap long lines whenever appropriate.
/approve

**Pull request merged**